### DISCUSSION TITLE
survey-multi-choice plugin in jsPsych 6.3.1

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2786] [Q&A] [answered]
Hi all,

I am using survey-multi-choice plugin in version 6.3.1, and I realized that the questions are NOT BOLDED, but the options are **bolded**. How can I overcome this? I just want to be able to format boldness in questions and options.

Thanks,
Fatih.

**Comment by ChristopheBossens** [comment, marked as answer]
Hi!

Can you share the code that you currently use to present the survey? I just checked it with 6.3.1 and nothing is bolded here by default. I can however apply a bold style by using the `<b></b>` tag in both the prompt as well as the choices:

```
{
  prompt: "<b>Which of the following do you like the least?</b>", 
  name: 'FruitDislike', 
  options: ['<b>Apple</b>', 'Banana', 'Orange', 'Grape', 'Strawberry'], 
}
```

**Reply by fspinar** [reply]
Hi @ChristopheBossens !

Here is an example question.
`var q7 = {
  type: 'survey-multi-choice',
  questions: [
    {
      prompt: 'How many cents does each coin worth?', 
      name: 'coinWorth', 
      options: ['10','1','0.1','0.01'],
    }, 
  ]
};`

I totally understand your example. But in my program here the options are bold on the screen, and I don't understand why...

Thank you!

**Reply by ChristopheBossens** [reply]
If you open the developer tools of your browser you should be able to inspect the element and it should also display an overview of the different styles that it has and where it got them from (e.g., which css class). 

Alternatively, can you try to set the options to a normal font weight like this:
`options: ["<span style='font-weight: 400'>10</span>",'1','0.1','0.01']`

**Reply by fspinar** [reply]
`options: ["<span style='font-weight: 400'>10</span>",'1','0.1','0.01']`

This worked! Thanks a lot @ChristopheBossens !
